libs/ui-components/src/lib/TodoList/TaskComponents/TaskModal.tsx

```diff
@@ -124,45 +124,43 @@ export const TaskModal: React.FC<TaskModalProps> = ({
                   <Thead>
                     <Tr>
                       <Th>Category:</Th>
-                      <Th>{task.category.name}</Th>
+                      <Th>{task?.category?.name}</Th>
```

Comment:
Instead, we can just do smth like:
if (!task) return null;

That's the convention of handling those scenarios.